package.json

```diff
@@ -991,6 +997,11 @@
           "command": "confluent.openCCloudLink",
           "when": "view in confluent.viewsWithResources && (viewItem in confluent.ccloudResources || viewItem =~ /ccloud-kafka-topic.*/ || viewItem =~ /.*ccloud.*schema$/)",
           "group": "z_openInCloud"
+        },
+        {
+          "command": "confluent.openCCloudApiKeysUrl",
```

Comment:
Let's make sure we have an entry for this command that prevents it from appearing in the command palette at all, so under `menus` and `commandPalette` let's add:
```
{
  "command": "confluent.openCCloudApiKeysUrl",
  "when": "false"
}
```
Current behavior doesn't throw any errors, but just as a bit of cleanup:
<img width="877" alt="image" src="https://github.com/user-attachments/assets/24e11571-172b-437c-be2e-9327539b59fa" />

Reply:
I think we need this for `confluent.openCCloudUrl` too since it looks like that does nothing either.

Reply:
Good catch! Mind doing that in this branch? If not, we can do a quick follow-up to handle it.

Reply:
Added entries for both